pom.xml

```diff
@@ -708,7 +710,7 @@
                 <artifactId>license-maven-plugin</artifactId>
                 <version>${license-maven-plugin.version}</version>
                 <configuration>
-                    <header>${jetcd.project.root}/etc/license.txt</header>
+                    <header>${session.executionRootDirectory}/etc/license.txt</header>
```

Comment:
@lburgazzoli does that look ok to you? I'm not an expert in maven, but with this change `./mvnw license:check` works locally for me.

Reply:
it does not look correct as that variable points to the path where the maven command was executed so if I go to a submodule, then the license check would not work.

That is the reason why the property jetcd.project.root exists, we should investigate why it didn’t get resolved.

Reply:
All right, thank you for explanation. I would revert it back then.